primap2/csg/_strategies/gaps.py

```diff
@@ -71,15 +68,6 @@ class FitParameters:
         minimal number of points to calculate the trend. Default is 1, but if the degree
         of the fit polynomial is higher than 1, the minimal number of data points
         the degree of the fit polynomial
-
```

Comment:
So we use docstrings in the methods and not a methods section. I'll try to remember in the future.

Reply:
Yeah, both works, but generally I like the docs to be as close to the code as possible.